testing/environments/5.0.0-cgroups.yml

```diff
@@ -0,0 +1,25 @@
+# This should test the environment with the latest snapshots
+# This is based on base.yml
+
+elasticsearch:
```

Comment:
I didn't use the make command to start it, as it creates more containers than I want, so the idea was to use the commands that are set in the description. Probably we also need to add -f ports.yml ...